components/layout/fragment.rs

```diff
@@ -1029,12 +1029,13 @@ impl Fragment {
     }
 
     /// Transforms this fragment into an ellipsis fragment, preserving all the other data.
-    pub fn transform_into_ellipsis(&self, layout_context: &LayoutContext) -> Fragment {
+    pub fn transform_into_ellipsis(&self, layout_context: &LayoutContext, text_overflow_string: String)
+    -> Fragment {
```

Comment:
nit: Instead of this alignment, let's use

```rust
pub fn transform_into_ellipsis(&self,
			       layout_context: &LayoutContext,
			       text_overflow_string: String)
			       -> Fragment {
```